src/arbitrary/letrec.ts

```diff
@@ -2,6 +2,11 @@ import { LazyArbitrary } from './_internals/LazyArbitrary';
 import { Arbitrary } from '../check/arbitrary/definition/Arbitrary';
 import { convertFromNext, convertToNext } from '../check/arbitrary/definition/Converters';
 
+export interface TieFunction<T> {
```

Comment:
- Add a small JSdoc, just a few words + the `@public` at the end of the block.
- Import and Re-Export it from src/fast-check-default.ts